misc/stock_emails.md

```diff
@@ -0,0 +1,111 @@
+
+#Inquiries about joining Mozilla L10n
+##General inquiry
+
+Hi [inquiree],
+
+Thank you for your email and welcome to the Mozilla Community! We're excited that you're interested in joining our localization (L10n) effort. 
+
+Joining our L10n effort is easy. Within five easy steps you can start contributing to the project! We have a wiki page specially designed to help you get started. Please visit http://wiki.mozilla.org/L10n:Contribute and follow the five steps to get involved.
+
+Could you also please provide me with some additional information about yourself, such as languages you speak, country you live in, and technical background?
+
+You can also visit the following wiki pages to learn the details of how we localize:
+* [L10n:Localization Process](https://wiki.mozilla.org/L10n:Localization_Process) A good description of the whole process.
+Looking forward to hearing from you,
+
+[name]
+
+###Follow up to general inquiry
+We're very excited to have you join the Mozilla localization (l10n) effort! Your skills and dedication will be very valuable to your l10n team.
+
+Next step, follow the steps outline on http://wiki.mozilla.org/L10n:Contribute. When you contact your l10n team, please CC me on that email so that I can help to make sure you receive a prompt response.
+
+If you have any questions or difficulties, please let me know.
+
+Welcome!
+
+##Starting a new localization
+Thank you for your email and welcome to the Mozilla Community! We're excited that you're interested in joining our localization effort. 
+
+Mozilla.org is usually the first place a user visits when they want to download Firefox. Because of that, we ask all new communities to begin with localizing the mozilla.org site. Since you have stated that you would like to start a new localization effort, [Peiying] (mailto:pmo@mozilla.com) will be your mentor and point of contact.
+
+Here's how you can get started:
+
+* Create an account on [Pontoon](https://pontoon.mozilla.org/teams).
+* Check our list of existing [L10n teams](https://pontoon.mozilla.org/teams) to see if an effort for your locale already exists. If an effort exists, contact the team manager to join the team and help drive the effort. If it doesn't exist, introduce your new L10n team on the [new locales newsgroup](http://groups.google.com/group/mozilla.dev.l10n.new-locales).
+* Introduce yourself to the L10n community on the [L10n newsgroup](http://groups.google.com/group/mozilla.dev.l10n.new-locales) by subscribing and posting an introduction. In your introduction, please add some brief information about yourself e.g., professional background, interests, etc.) and what you hope to work on (e.g., "I saw that there was xx-XX localization team and I'd like to help them out.").
```

Comment:
Did you want to use just dev-l10n here? You have a missing parenthesis before the first e.g.